x-pack/plugins/ml/public/application/contexts/ml/ml_context.ts

```diff
@@ -8,18 +8,15 @@
 import React from 'react';
 import type { DataView, DataViewsContract } from '@kbn/data-views-plugin/public';
 import type { SavedSearch } from '@kbn/saved-search-plugin/public';
-import type { SavedSearchSavedObject } from '../../../../common/types/kibana';
+import type { IUiSettingsClient } from '@kbn/core/public';
 import type { MlServicesContext } from '../../app';
 
 export interface MlContextValue {
   combinedQuery: any;
-  currentDataView: DataView; // TODO this should be DataView or null
-  // @deprecated currentSavedSearch is of SavedSearchSavedObject type, change to selectedSavedSearch
-  deprecatedSavedSearchObj: SavedSearchSavedObject | null;
+  selectedDataView: DataView; // // TODO this should be DataView or null
```

Comment:
Should this type be updated or can that comment be removed?

Reply:
Removed the comment here [`bfcee95` (#159070)](https://github.com/elastic/kibana/pull/159070/commits/bfcee95d4cd8af9193f61f0e7b9412851775f9f5)